pkg/helm/handlers/handler_test.go

```diff
@@ -0,0 +1,188 @@
+package handlers
+
+import (
+	"errors"
+	"helm.sh/helm/v3/pkg/action"
+	"net/http"
+	"net/http/httptest"
+	"strings"
+	"testing"
+
+	"github.com/openshift/console/pkg/auth"
+	"helm.sh/helm/v3/pkg/release"
+)
+
+var fakeReleaseList = []*release.Release{
+	{
+		Name: "Test",
+	},
+}
+
+var fakeRelease = release.Release{
+	Name: "Test",
+}
+
+var fakeReleaseManifest = "manifest-data"
```

Comment:
Do you want this to semi-resemble a real response?

Reply:
not necessarily -just want to check if the provided input generate appropriate output.